pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -243,6 +244,21 @@ func (m *kubeGenericRuntimeManager) Type() string {
 	return m.runtimeName
 }
 
+// SupportsSingleFileMapping returns whether the container runtime supports single file mappings or not.
+// It is supported on Windows only if the container runtime is containerd.
+func (m *kubeGenericRuntimeManager) SupportsSingleFileMapping() bool {
+	switch goruntime.GOOS {
+	case "windows":
+		{
```

Comment:
Don't really need the brackets. 
Same below.

Reply:
Done